airflow/models/baseoperator.py

```diff
@@ -1691,6 +1693,16 @@ def inherits_from_empty_operator(self):
         # of its subclasses (which don't inherit from anything but BaseOperator).
         return getattr(self, "_is_empty", False)
 
+    @property
+    def start_trigger(self) -> BaseTrigger | None:
+        """Trigger when deferring task."""
+        return getattr(self, "_start_trigger", None)
+
+    @property
+    def next_method(self) -> str | None:
+        """Method to execute after finish deferring."""
+        return getattr(self, "_next_method", None)
```

Comment:
Why do we not need to do the same for this attribute?

Reply:
This is not yet set in `__init__`, but yep, this could be moved to `__init__` and changed into `return self._next_method`